api/src/main/resources/liquibase.xml

```diff
@@ -576,4 +594,10 @@
             <column name="description" value="URL pattern to use for published Recurring appointment events. Default is /openmrs/ws/rest/v1/recurring-appointments?uuid={uuid}. If you change default value, please add your custom implementation for the given URL"/>
         </insert>
     </changeSet>
+    <changeSet id="default-202009011116" author="Deedee Lee">
```

Comment:
add precondition to check whether email as person attribute type exists

Reply:
Done